wave_vision/CMakeLists.txt

```diff
@@ -4,40 +4,52 @@ PROJECT(wave_vision)
 # DEPENDENCIES
 FIND_PACKAGE(OpenCV REQUIRED)
 FIND_PACKAGE(Eigen3 REQUIRED)
+FIND_PACKAGE(Boost 1.40 COMPONENTS system filesystem REQUIRED)
```

Comment:
@chutsu in fact you _should_ use boost filesystem, no point maintaining our own non-cross-platform filesystem code.